UPGRADE-4.0.md

```diff
@@ -1,6 +1,10 @@
 UPGRADE FROM 3.x to 4.0
 =======================
 
+## AdminInterface
+
+If you're using a custom implementation of `AdminInterface`, you need to add realisation of `hasAccess` method.
```

Comment:
We already have a note about added method on interface here: https://github.com/sonata-project/SonataAdminBundle/pull/3755/files#diff-b2c918cba71bea146d3145ca75754b85R18

Could you complete this part instead? Thanks.